src/main/java/server/router/ServerRouter.java

```diff
@@ -0,0 +1,23 @@
+package main.java.server.router;
+
+import main.java.handlers.DefaultHandler;
+import main.java.handlers.Handler;
+import main.java.handlers.HandlerCollector;
+import main.java.server.Response;
+import main.java.server.request.Request;
+
+public class ServerRouter implements Router {
+
+    @Override
+    public Response route(Request request) {
+        Route route = new Route(request.getRequestMethod(), request.getRequestPath());
+        var allHandlers = HandlerCollector.ALL_HANDLERS;
+
+        for (Handler handler : allHandlers) {
```

Comment:
I wonder if there's a functional approach that would enable us to lookup the appropriate handler without iterating through a `for` loop 🤔